src/Kdyby/Doctrine/EntityManager.php

```diff
@@ -281,6 +284,68 @@ public function onDaoCreate(EntityManager $em, Doctrine\Common\Persistence\Objec
 
 
 
+	/****************** Kdyby\Persistence\QueryExecutor *****************/
+
+
+
+	/**
+	 * @param \Kdyby\Persistence\Query|\Kdyby\Doctrine\QueryObject $queryObject
+	 * @param int $hydrationMode
+	 * @throws QueryException
+	 * @return array|\Kdyby\Doctrine\ResultSet
+	 */
+	public function fetch(Persistence\Query $queryObject, $hydrationMode = AbstractQuery::HYDRATE_OBJECT)
```

Comment:
Now let's hope doctrine won't decide to create a `fetch` method for something here :)